core/src/test/java/io/confluent/kafka/schemaregistry/RestApp.java

```diff
@@ -73,6 +73,7 @@ public void start() throws Exception {
     restConnect = restServer.getURI().toString();
     if (restConnect.endsWith("/"))
       restConnect = restConnect.substring(0, restConnect.length()-1);
+    // restClient = new RestService(restConnect, false, true);
```

Comment:
I assume this is for testing the Apache http client.  Maybe add a comment if you want to keep this commented out?